CallsheetLocalizations/pl.xcloc/Localized Contents/pl.xliff

```diff
@@ -978,14 +1034,17 @@
       </trans-unit>
       <trans-unit id="Parental guidance won't be shown for cast and crew." xml:space="preserve">
         <source>Parental guidance won't be shown for cast and crew.</source>
+        <target>Ograniczenie wiekowe nie będzie pokazywane dla obsady i ekipy.</target>
         <note>Shown in Quick Access settings when the user selects Parental Guidance as their Quick Access button</note>
       </trans-unit>
       <trans-unit id="Persnickety Preferences" xml:space="preserve">
         <source>Persnickety Preferences</source>
+        <target>Wybór wybredny</target>
```

Comment:
"Opcje dla wybrednych"? Sam nie wiedziałem jak to przetłumaczyć, ale tak będzie chyba najlepiej.